src/components/ArticleComponents/AuthorList/Author/index.tsx

```diff
@@ -18,6 +19,10 @@ const Author = ({
   const githubUserName = username.trim();
   const githubLink = `https://github.com/${githubUserName}`;
   const githubImgLink = `https://github.com/${githubUserName}.png?size=${size}`;
+  
+  const authorImg = {
+    backgroundImage: `url(${githubImgLink}), url(${authorPlaceholderImg})`,
+  };
 
   // If it's the first author then no margin left.
   const mleft = index === 0 ? { marginLeft: 0 } : {};
```

Comment:
Noticed that margin left is applied `0` in stylesheet,

https://github.com/nodejs/nodejs.dev/blob/7dc505a2e4b177231c0a95532c1f3745a4cb4aed/src/components/ArticleComponents/AuthorList/index.module.scss#L19

`mleft` is redundant, Shall I remove it?

Reply:
Try to delete it, if it doesn't change the look let's go. If not we leave it like that. 😂

Reply:
It didn't change :)